ApptentiveFeedback/ApptentiveFeedback/Additions/ApptentiveCallbacks.cs

```diff
@@ -0,0 +1,350 @@
+        public static void CanShowMessageCenter(Action<bool> onCompletion)
+        {
+            ApptentiveKit.Android.Apptentive.CanShowMessageCenter(new BooleanCallback(onCompletion));
+        }
+
+        /// <summary>
+        /// Sets the user's name. This name will be sent to the Apptentive server and displayed in
+        /// conversations you have with this person.This name will be the definitive username for this
+        /// user, unless one is provided directly by the user through an Apptentive UI.
+        /// </summary>
+        /// <param name="name">The name of the user.</param>
+        public static void SetPersonName(String name)
+        {
+            ApptentiveKit.Android.Apptentive.PersonName = name;
+
+        }
+
+        /// <summary>
+        /// Retrieves the user's name.
+        /// </summary>
+        /// <returns>The user's name if set.</returns>
+        public static string GetPersonName()
+        {
+            return ApptentiveKit.Android.Apptentive.PersonName;
+        }
+
+        /// <summary>
+        /// Sets the user's email address. This email address will be sent to the Apptentive server to
+        /// allow out of app communication, and to help provide more context about this user.This email
+        /// will be the definitive email address for this user, unless one is provided directly by the
+        /// user through an Apptentive UI.
+        /// </summary>
+        /// <param name="email">The email of the user.</param>
+        public static void SetPersonEmail(String email)
+        {
+            ApptentiveKit.Android.Apptentive.PersonEmail = email;
+        }
+
+        /// <summary>
+        /// Retrieves the user's email.
+        /// </summary>
+        /// <returns>The email of the user if set.</returns>
+        public static string GetPersonEmail()
+        {
+            return ApptentiveKit.Android.Apptentive.PersonEmail;
+        }
+
+        /// <summary>
+        /// Add a custom data String to the Person. Custom data will be sent to the server, is displayed
+        /// in the Conversation view, and can be used in Interaction targeting.
+        /// </summary>
+        /// <param name="key">The key to store the data under.</param>
+        /// <param name="value">A String value.</param>
+        public static void AddCustomPersonData(String key, String value)
+        {
+            ApptentiveKit.Android.Apptentive.AddCustomPersonData(key, value);
+        }
+
+        /// <summary>
+        /// Add a custom data int to the Person. Custom data will be sent to the server, is displayed
+        /// in the Conversation view, and can be used in Interaction targeting.
+        /// </summary>
+        /// <param name="key">The key to store the data under.</param>
+        /// <param name="value">A int value.</param>
+        public static void AddCustomPersonData(String key, int value)
```

Comment:
Would this only support int?

Reply:
It takes an int from the app but converts to Number for the sdk.

Reply:
What did legacy do? We are supporting all the Number format inputs (Int, Long, Float and Double) in native and plugins.

Reply:
Luqmaan has removed this entirely from the wrapper and now it is available for all the numeric types. He is to verify this in XoloFacts.